core/arch/arm/include/mm/mobj.h

```diff
@@ -39,6 +39,7 @@
 struct mobj {
 	const struct mobj_ops *ops;
 	size_t size;
+	size_t granule;
```

Comment:
Okay, `phys_granule` is better name. There are `mobj_get_granule()` function that returns corresponding value.